wagtail/admin/templates/wagtailadmin/login.html

```diff
@@ -31,29 +31,24 @@ <h1>{% block branding_login %}{% trans "Sign in to Wagtail" %}{% endblock %}</h1
 
                 {% block fields %}
                     <div class="login-fields">
-                        {% field field=form.username icon="user" sr_only_label=True %}{% endfield %}
-                        {% field field=form.password icon="password" sr_only_label=True %}{% endfield %}
+                        {% field field=form.username %}{% endfield %}
+                        {% field field=form.password %}{% endfield %}
 
                         {% block extra_fields %}
                             {% for field_name, field in form.extra_fields %}
                                 {% field field=field %}{% endfield %}
                             {% endfor %}
                         {% endblock extra_fields %}
+                        {% trans "Remember me" as checkbox_text %}
+                        {% include "wagtailadmin/shared/forms/single_checkbox.html" with name="remember" label_classname="w-label=3" text=checkbox_text %}
+                        {% if show_password_reset %}
+                            <a class="reset-password" href="{% url 'wagtailadmin_password_reset' %}" aria-label="{% trans 'Reset your password' %}">{% trans "Forgot your password?" %}</a>
+                        {% endif %}
                     </div>
-
-                    {% trans "Remember me" as checkbox_text %}
-                    {% include "wagtailadmin/shared/forms/single_checkbox.html" with name="remember" label_classname="w-label=3 w-mb-4" text=checkbox_text %}
                 {% endblock %}
-
-                <div class="w-mt-4 w-mb-4">
-                    {% block submit_buttons %}
-                        <button type="submit" class="button button-longrunning" data-clicked-text="{% trans 'Signing in…' %}">{% icon name="spinner" %}<em>{% trans 'Sign in' %}</em></button>
-                    {% endblock %}
-                </div>
-
-                {% if show_password_reset %}
-                    <a class="reset-password" href="{% url 'wagtailadmin_password_reset' %}" aria-label="{% trans 'Reset your password' %}">{% trans "Forgotten password?" %}</a>
```

Comment:
best to keep the original translation - even though the design had a different label - I think this can be left as is for the sake of translations and it's not really a design decision.

Reply:
Cool cool